spec/controllers/councils_controller_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+describe CouncilsController do
+  let(:user) {FactoryGirl.create :user}
+  let(:council) { FactoryGirl.create :council }
+
+  before do
+    stub_current_user user
+  end
+
+  describe "#index" do
+    it "assigns the councils to Council.all" do
+      get :index
+      expect(response).to be_success
+    end
+  end
+
+  describe "#new" do
+    it "assigns the council to Council.new " do
+      get :new
+      expect(assigns(:council)).to be_a_new(Council)
+    end
+  end
+
+  describe "#create" do
+    it "creates the council if valid params" do
+      post :create, council: FactoryGirl.attributes_for(:council)
+
```

Comment:
You might want to check the council actually got created. That's what the test claims to do.